internal/api/cluster_installation.go

```diff
@@ -24,6 +25,7 @@ func initClusterInstallation(apiRouter *mux.Router, context *Context) {
 	clusterInstallationRouter.Handle("", addContext(handleGetClusterInstallation)).Methods("GET")
 	clusterInstallationRouter.Handle("/config", addContext(handleGetClusterInstallationConfig)).Methods("GET")
 	clusterInstallationRouter.Handle("/config", addContext(handleSetClusterInstallationConfig)).Methods("PUT")
+	clusterInstallationRouter.Handle("/exec/{command}", addContext(handleRunClusterInstallationExecCommand)).Methods("POST")
 	clusterInstallationRouter.Handle("/mattermost_cli", addContext(handleRunClusterInstallationMattermostCLI)).Methods("POST")
```

Comment:
Goal is to eventually migrate this to be under `/exec/{command}` too, right?

Reply:
Yes, exactly, just didn't want to break the old endpoint yet.